XIV on Mac/de.lproj/Main.strings

```diff
@@ -633,112 +633,112 @@
 "ngK-lv-alc.title" = "Add On";
 
 /* Class = "NSMenuItem"; title = "Troubleshooting…"; ObjectID = "nwX-vr-aWN"; */
-"nwX-vr-aWN.title" = "Troubleshooting…";
+"nwX-vr-aWN.title" = "Fehlersuche...";
 
 /* Class = "NSButtonCell"; title = "Next"; ObjectID = "o4g-vh-B1L"; */
-"o4g-vh-B1L.title" = "Next";
+"o4g-vh-B1L.title" = "Nächste";
 
 /* Class = "NSTabViewItem"; label = "Copy game folder"; ObjectID = "o5w-vX-Qqm"; */
-"o5w-vX-Qqm.label" = "Copy game folder";
+"o5w-vX-Qqm.label" = "Spieleordner kopieren";
 
 /* Class = "NSButtonCell"; title = "Esync"; ObjectID = "o9I-0w-Ncp"; */
 "o9I-0w-Ncp.title" = "Esync";
 
 /* Class = "NSWindow"; title = "Troubleshooting"; ObjectID = "oJM-9v-5CI"; */
-"oJM-9v-5CI.title" = "Troubleshooting";
+"oJM-9v-5CI.title" = "Fehlersuche";
 
 /* Class = "NSButtonCell"; title = "Frame Rate"; ObjectID = "olm-gB-DgF"; */
-"olm-gB-DgF.title" = "Frame Rate";
+"olm-gB-DgF.title" = "Bildrate";
 
 /* Class = "NSMenuItem"; title = "Delete"; ObjectID = "pa3-QI-u2k"; */
-"pa3-QI-u2k.title" = "Delete";
+"pa3-QI-u2k.title" = "Löchen";
 
 /* Class = "NSButtonCell"; title = "Next"; ObjectID = "phh-fm-6OL"; */
-"phh-fm-6OL.title" = "Next";
+"phh-fm-6OL.title" = "Nächste";
 
 /* Class = "NSTextFieldCell"; title = "If your game is freezing at startup, \"Compiling Shaders…\" forever, missing lighting or randomly freezing, the problem may be with your shader cache. Deleting it will cause a new one to be rebuilt, though it may result in occasional stutters during play initially."; ObjectID = "piU-pX-vSl"; */
-"piU-pX-vSl.title" = "If your game is freezing at startup, \"Compiling Shaders…\" forever, missing lighting or randomly freezing, the problem may be with your shader cache. Deleting it will cause a new one to be rebuilt, though it may result in occasional stutters during play initially.";
+"piU-pX-vSl.title" = "Wenn dein Spiel beim Start einfriert, \"Compiling Shaders...\" ewig dauert, die Beleuchtung fehlt oder das Spiel zufällig einfriert, liegt das Problem möglicherweise an deinem Shader-Cache. Wenn du ihn löschst, wird ein neuer Cache erstellt, auch wenn dies anfangs zu gelegentlichen Stottern beim Spielen führen kann.";
 
 /* Class = "NSMenuItem"; title = "Find Next"; ObjectID = "q09-fT-Sye"; */
-"q09-fT-Sye.title" = "Find Next";
+"q09-fT-Sye.title" = "Nächste finden";
 
 /* Class = "NSButtonCell"; title = "Launch Vanilla Game"; ObjectID = "qDd-CO-Fjs"; */
-"qDd-CO-Fjs.title" = "Launch Vanilla Game";
+"qDd-CO-Fjs.title" = "Vanille-Spiel starten";
```

Comment:
> "Unmodifiziertes Spiel starten" 
is probably what you want to use here, for use Vanille is mainly something we associate with ice cream :D